### ISSUE TITLE
Support for op version 2 ?

### ISSUE THREAD

**Issue by sdahlbac** [anasinnyk/terraform-provider-1password#64]
**Is your feature request related to a problem? Please describe.**

Op version 2 seems to have changed the commandline arguments. 

https://developer.1password.com/docs/cli/upgrade/

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by anasinnyk** [comment]
Hi @sdahlbac I think we can use API for that. When I started this project 1Password didn't have API. But I don't have any time for implement that. Sorry.

**Comment by sdahlbac** [comment]
You mean the one needing secrets automation? If so, then I presume the official 1password provider is the way to go?

Based on the docs of op version 2 (specifically the bit about the binary having to be called exactly /usr/local/bin/op) I thought that this provider using op v1 would block cli usage of op v2. However, it does not seem to be that black and white. Op v2 runs fine even called op2, but possibly auto updates might not work?

**Comment by anasinnyk** [comment]
Yes I think better way uses this official provider https://registry.terraform.io/providers/1Password/onepassword

**Comment by stopthatastronaut** [comment]
Ah, I assume this is why I'm getting 

`Error: Cannot signin: [ERROR] 2022/09/21 10:01:53 unknown flag: --output`

**Comment by sdahlbac** [comment]
As a simpler solution, could we first check eg. /usr/local/bin/op1 and use that if it exists. That way one could have the new op v2 installed in the expected location and this provider could still work with op version 1?
